.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: build
+
+on:
+  push:
+    branches: [dev]
+    paths-ignore:
+      - "docs/**"
+      - "readme.md"

```

Comment:
Use the same casing as the actual files/folders for consistency even if it works. Same for HE.

```suggestion
      - "Docs/**"
      - "Readme.md"

```